Cargo.lock

```diff
@@ -136,21 +137,15 @@ dependencies = [
 
 [[package]]
 name = "bitflags"
-version = "1.3.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
-
-[[package]]
-name = "bitflags"
-version = "2.4.2"
+version = "2.5.0"
```

Comment:
There's quite a lot of package bumps here, including to some major package versions. We feel pretty confident in bumping all these right now?

Reply:
Confident enough… this is all from cargo update.